README.md

```diff
@@ -144,6 +145,7 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Telnyx](https://github.com/novuhq/novu/tree/main/providers/telnyx)
 - [x] [Termii](https://github.com/novuhq/novu/tree/main/providers/termii)
 - [x] [Gupshup](https://github.com/novuhq/novu/tree/main/providers/gupshup)
+- [x] [MailerSend](https://github.com/novuhq/novu/tree/main/providers/mailersend)
```

Comment:
@Agathe-Brusset I think mailersend only doing email right?

Reply:
Hey @scopsy ! No, our API also allows sending SMS to US numbers.

Reply:
Unfortunately, the integration done is just for the email functionalities, so we can't add it to the SMS providers. 😞

Reply:
Oh okay then, we can drop this line :D

Reply:
We will try in the future or with the support of the community to integrate the SMS functionalities, but that might require some work on our side as currently the provider IDs are unique and can't be repeated in integrations for different channels, such this case.